$OpenBSD: patch-rights-paths,v 1.1 2000/03/28 11:53:03 camield Exp $

This port wants to change the rights on PREFIX/ and PREFIX/bin to 02755.
Usually PREFIX is /usr/local. The rights on /usr/local and /usr/local/bin
should be 0755 according to /etc/mtree/4.4BSD.dist.

Also, use share/doc instead of doc.

--- hier.c.orig	Sun Nov 15 03:28:23 1998
+++ hier.c	Wed Dec 29 19:28:59 1999
@@ -2,14 +2,15 @@
 
 void hier()
 {
-  h(auto_home,-1,-1,02755);
+  h(auto_home,-1,-1,0755);
 
-  d(auto_home,"bin",-1,-1,02755);
-  d(auto_home,"doc",-1,-1,02755);
-  d(auto_home,"doc/serialmail",-1,-1,02755);
-  d(auto_home,"man",-1,-1,02755);
-  d(auto_home,"man/man1",-1,-1,02755);
-  d(auto_home,"man/cat1",-1,-1,02755);
+  d(auto_home,"bin",-1,-1,0755);
+  d(auto_home,"man",-1,-1,0755);
+  d(auto_home,"man/man1",-1,-1,0755);
+  d(auto_home,"man/cat1",-1,-1,0755);
+  d(auto_home,"share",-1,-1,0755);
+  d(auto_home,"share/doc",-1,-1,0755);
+  d(auto_home,"share/doc/serialmail",-1,-1,0755);
 
   c(auto_home,"bin","serialqmtp",-1,-1,0755);
   c(auto_home,"bin","serialsmtp",-1,-1,0755);
@@ -31,7 +32,7 @@
   c(auto_home,"man/man1","setlock.1",-1,-1,0644);
   c(auto_home,"man/cat1","setlock.0",-1,-1,0644);
 
-  c(auto_home,"doc/serialmail","FROMISP",-1,-1,0644);
-  c(auto_home,"doc/serialmail","TOISP",-1,-1,0644);
-  c(auto_home,"doc/serialmail","AUTOTURN",-1,-1,0644);
+  c(auto_home,"share/doc/serialmail","FROMISP",-1,-1,0644);
+  c(auto_home,"share/doc/serialmail","TOISP",-1,-1,0644);
+  c(auto_home,"share/doc/serialmail","AUTOTURN",-1,-1,0644);
 }
